css/css-inline/text-box-trim/text-box-trim-half-leading-block-box-002-ref.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<title>Reference for not trimming block-boxes at their first/last formatted lines if the FCL has non-zero padding. </title>
+<link rel="help" href="https://drafts.csswg.org/css-inline-3/#leading-trim">
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css" />
+
+<style>
+.div-parent {
+  outline: 1px solid orange;
+  font-family: Ahem;
+  font-size: 20px;
+  line-height: 2;
+}
+</style>
+
+<div class ="div-parent"">
```

Comment:
Remove an excessive double-quotes at the end.

Reply:
thanks! fixed.